tests/scripts/all.sh

```diff
@@ -1098,16 +1098,24 @@ helper_psasim_server() {
     if [ "$OPERATION" == "start" ]; then
     (
         cd tests
-        msg "start server"
+        msg "start server in tests"
         psa-client-server/psasim/test/start_server.sh
+        msg "start server in tf-psa-crypto/tests"
+        cd ../tf-psa-crypto/tests
+        ../../tests/psa-client-server/psasim/test/start_server.sh
```

Comment:
Should we just do one of these, depending what directory/directories exist (`-d foo`)?

Reply:
No we need both. With this PR we now have x509 and TLS test suites in `tests` and the crypto test suites in `tf-psa-crypto/tests`. When in `test_suite_with_psasim` we call `make PSASIM=1 test` we first execute the test suites located in `tests` and for these ones we need a PSA server started from `tests` and then we run the test suites located in `tf-psa-crypto/tests` for which we need a PSA server started from `tf-psa-crypto/tests`.

Reply:
Ah okay, thanks